src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1251,14 +1251,59 @@ void TwitchMessageBuilder::appendTwitchBadges()
                 break;
             }
         }
+        else if (badge.startsWith("founder/"))
+        {
+            if (auto badgeEmote =
+                    this->twitchChannel->globalTwitchBadges().badge("founder",
+                                                                    "0"))
+            {
+                auto badgeInfo = this->tags.find("badge-info");
+                if (badgeInfo != this->tags.end() &&
+                    badgeInfo.value().toString().split(',')[0].startsWith(
+                        "founder/"))
+                {
+                    auto subMonths =
+                        badgeInfo.value().toString().split(',')[0].mid(8);
+                    this->emplace<BadgeElement>(
+                            badgeEmote.get(),
+                            MessageElementFlag::BadgeSubscription)
+                        ->setTooltip(QString((*badgeEmote)->tooltip.string) +
+                                     " (" + subMonths + " months)");
```

Comment:
It can be simplified with `QString().arg()`.
Something like this.
```
QString("%1 (%2 months)").arg((*badgeEmote)->tooltip.string).arg(subMonths));
```